src/eth.rs

```diff
@@ -101,6 +101,27 @@ pub struct Transaction {
     pub data: Vec<u8>,
 }
 
+#[cfg_attr(
+    feature = "wasm",
+    derive(serde::Deserialize),
+    serde(rename_all = "camelCase")
+)]
+pub struct EIP1559Transaction {
+    pub decoded_chain_id: Vec<u8>,
```

Comment:
why not `chain_id: u64`? it is `u64` pretty much everywhere I think

Reply:
Because the current implementation of the rlp decoding returns all fields as Vec<u8>:
```
let [decoded_chain_id, nonce, max_priority_fee_per_gas, max_fee_per_gas, gas_limit, recipient, value, data, _, _, _]: [Vec<u8>; 11]
```

I cast it as u64 in eth_sign_1559_transaction

Reply:
yeah, good point about the chainIDs bigger than 1 byte, duh 🤦‍♂️ 
added the sandbox signing and fixed here (tested with large chainIDs): https://github.com/digitalbitbox/bitbox-api-rs/pull/59/commits/6ad12788896325c90740a00801186c9410617d92